README.md

```diff
@@ -1,3 +1,10 @@
-# basic-training
-We will learn how to code in Python
-yes we do!
+-basic bash
```

Comment:
please use markdown syntax, check : https://guides.github.com/features/mastering-markdown/ for instructions